### ISSUE TITLE
Error: zmap: unrecognized option '--blacklist-file=/etc/zmap/blacklist.conf'

### ISSUE THREAD

**Issue by sundruid** [zmap/zmap#894]
**Describe the bug**
'blacklist' seems to be used internally somewhere. I see that this was changed to use 'blocklist'? Anyone having the same issue on a clean compile?

**CLI Arguments**
zmap --version
zmap -p 80
zmap <literally anything>

```
(zmap: unrecognized option '--blacklist-file=/etc/zmap/blacklist.conf')
```

**Comment by phillip-stephens** [comment]
Hey @sundruid, thanks for opening this issue!
Yep, the `--blacklist-file` option was changed to `--blocklist-file` but the usage should be identical.

New installs have (just tested this to be sure) the blocklist file installed at `/etc/zmap/blocklist.conf`, but since it seems you installed ZMap prior to the flag name change, if you could run `mv /etc/zmap/blacklist.conf /etc/zmap/blocklist.conf` everything should work correctly.

**Comment by phillip-stephens** [comment]
Just realized that my previous comment won't fully solve things for you.

Can you edit your `/etc/zmap/zmap.conf` file, replacing:
`blacklist-file "/etc/zmap/blacklist.conf"`

with

`blocklist-file "/etc/zmap/blocklist.conf"`

That should solve your issue. Looking at an easier fix for this upgrade path in #895.

**Comment by sundruid** [comment]
Perfect.. works. Thank you Phillip.
